src/js/vic-v2/components/DD214Description.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default function DD214Description() {
+  return (
+    <div>
+      <p>Please upload a copy of your DD 214 to verify that you were honorably discharged.</p>
```

Comment:
It looks like this is also spelled DD214 elsewhere on the site, should we use that spelling here as well?